app/Http/Controllers/Forum/TopicsController.php

```diff
@@ -415,6 +432,32 @@ public function show($id)
         ));
     }
 
+
+    /**
+     * Create Topic
+     *
+     * Create a new topic.
+     *
+     * ---
+     *
+     * ### Response Format
+     *
+     * Field  | Type                       | Includes
+     * ------ | -------------------------- | --------
+     * topic  | [ForumTopic](#forum-topic) | |
+     * post   | [ForumPost](#forum-post)   | body
```

Comment:
I missed it in the previous one, but these should match, either `[ForumTopic](#forumtopic)` or `[Forum Topic](#forum-topic)`.